src/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.builds

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Security.Cryptography.Pkcs.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+    </Project>
+    <Project Include="System.Security.Cryptography.Pkcs.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netcore50</TargetGroup>
+    </Project>
+    <Project Include="System.Security.Cryptography.Pkcs.csproj">
+      <OSGroup>Unix</OSGroup>
+    </Project>
+    <Project Include="System.Security.Cryptography.Pkcs.csproj">
+      <TargetGroup>net46</TargetGroup>
+    </Project>
+    <Project Include="System.Security.Cryptography.Pkcs.csproj">
+      <TargetGroup>net461</TargetGroup>
```

Comment:
X509Certificates has a different contract for net46 (no ECDsa) and net461 (yes ECDsa), so it needs to build two separate facades.  I don't expect that will apply here, since ECDsa shouldn't be specifically part of the surface area here.